controls/radgridview/hierarchical-gridview/hierachy-overview.md

```diff
@@ -12,11 +12,21 @@ position: 0
 
 Besides the trivial tabular data visualization, the __RadGridView__ is capable of displaying hierarchical data in the form of nested tables. In this case the rows of the __RadGridView__ are represented by the __GridViewExpandableRow__ class. In the [Basic Hierarchies]({%slug gridview-basic-hierarchies%}) topic you can find information about the basics of building a hierarchical grid.
```

Comment:
It seems that the GridViewExpandableRow class has been removed a long time ago. We should remove it from the article.